README.md

```diff
@@ -1,2 +1,401 @@
+
+- Get the source code with Zephyr module ZXA as the `west`
+  manifest repository
+- Build and run projects
+
+
+## Get Zephyr and Install Python Dependencies
+
+Clone Zephyr module ZXA and its dependencies into a new `west`
+workspace named :file:`zp`. You'll also install additional Python
+dependencies.
+
+### Ubuntu
+
+1. Get the Zephyr source code:
+
+      ```bash
+      west init \
+        -m https://github.com/zephyrproject-rtos/example-application \
+        --mr main \
+        --mf zephyr/west.yml \
+        ~/zxa
+      cd ~/zxa
+      west update
+      source zephyr/zephyr-env.sh
+      ```
+
+1. Export a `Zephyr CMake package`. This allows CMake to
+   automatically load boilerplate code required for building Zephyr
+   applications.
+
+      ```bash
+      west zephyr-export
+      ```
+
+1. Install additional Python dependencies specified by any module
+
+      ```bash
+      pip3 install --user -r ~/zxa/zephyr/scripts/requirements.txt
+      ```
+
+1. Configure `west` to place build output in 
+   project-board-specific folders
+
+      ```bash
+      west config \
+        build.dir-fmt ${ZEPHYR_BASE}/../build/{source_dir}/{board}
+      ```
+
+### macOS
+
+1. Get the Zephyr source code:
+
+      ```bash
+      west init \
+        -m https://github.com/zephyrproject-rtos/example-application \
+        --mr main \
+        --mf zephyr/west.yml \
+        ~/zxa
+      cd ~/zxa
+      west update
+      source zephyr/zephyr-env.sh
+      ```
+
+1. Export a `Zephyr CMake package`. This allows CMake to
+   automatically load boilerplate code required for building Zephyr
+   applications.
+
+      ```bash
+      west zephyr-export
+      ```
+
+1. Install additional Python dependencies specified by any module
+
+      ```bash
+      pip3 install -r ~/zxa/zephyr/scripts/requirements.txt
+      ```
+
+1. Configure `west` to place build output in 
+   project-board-specific folders
+
+      ```bash
+      west config \
+        build.dir-fmt ${ZEPHYR_BASE}/../build/{source_dir}/{board}
+      ```
+
+### Windows
+
+1. Get the Zephyr source code:
+
+      ```bat
+      cd %HOMEPATH%
+      west init \
+        -m https://github.com/zephyrproject-rtos/example-application \
+        --mr main \
+        --mf zephyr\west.yml \
+        zxa
+      cd zxa
+      west update
+      source zephyr\zephyr-env.sh
+      ```
+
+1. Export a `Zephyr CMake package`. This allows CMake to
+   automatically load boilerplate code required for building Zephyr
+   applications.
+
+      ```bat
+      west zephyr-export
+      ```
+
+1. Install additional Python dependencies specified by any module
+
+      ```bat
+      pip3 install -r HOMEPATH%\zxa\zephyr\scripts\requirements.txt
+      ```
+
+1. Configure `west` to place build output in 
+   project-board-specific folders
+
+      ```bat
+      west config build.dir-fmt ${ZEPHYR_BASE}\..\build\{source_dir}\{board}
+      ```
+
+
+## Build and Run the zxa_example_project
```

Comment:
Same here, we don't need a guide per operating system. A single one is more than enough if you ask users to place themselves inside the `example-application` folder